components/DetailsHeader/DetailsHeader.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="details-header">
+    <div class="details-header__container">
+      <div class="details-header__container--image">
+        <slot name="banner image" />
+      </div>
+      <div class="details-header__container--content">
+        <slot name="subtitle" />
+        <slot name="title" />
+        <slot name="description" />
+        <slot name="meta content" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'DetailsHeader'
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../../assets/_variables.scss';
+.details-header {
+  margin: 1.25rem;
+  &__container {
+    display: flex;
+    flex-direction: row;
+    border: solid 1px $cloudy;
+    padding: 1rem;
+    padding-top: 0;
+    background: white;
+    &--image {
+      margin-right: 1rem;
+      margin-top: 2rem;
+    }
+    &--content {
```

Comment:
Here, the code is styling elements, but since we're using slots, these elements are not guaranteed to exist. This is fine for now because we're controlling everything, but once SPARC starts to maintain this repo, it may not function like we intended. If we're just displaying strings, and we need them to be specific elements, a prop may be better.

Reply:
So you're recommending all the slots under `details-header__container--content` be changed to props instead?

Reply:
The problem with this is that each header has a different style for the text. For example, the text on the project details page is larger for the description than that of the dataset details page. So I would need to style those specific elements differently across pages, which may be difficult to do so if I just passed them into the component as regular props.

Reply:
Actually, on second thought, I may be able to do this for just the `Subtitle`, `Title`, and `Description`. I'll address the fix.

Reply:
My recommendation is to change any common data, and ones that are styled to a specific tag, to a prop. Your suggestions of changing `subtitle`,`title` and `description` sound great.